client/components/App.jsx

```diff
@@ -34,14 +45,43 @@ const App = () => {
       .then((productRes) => {
         // console.log(productRes.data[0]);
         setProduct(productRes.data[0]);
-        axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sea/products/${productRes.data[0].id}/styles`, {
+        // get the styles data from the default product id
+        axios.get(`${url}products/${productRes.data[0].id}/styles`, {
           headers: {
             Authorization: TOKEN,
           },
         })
           .then((styleRes) => {
-            // console.log(styleRes.data.results);
+            // console.log(styleRes);
             setStyles(styleRes.data.results);
+            // get the reviews meta data from the default product id
+            axios.get(`${url}reviews/meta?product_id=${productRes.data[0].id}`, {
```

Comment:
All the axios.get statements can be in the .then() body at line 45. This is so that all the fetch operations can run at the same time instead of waiting for the subsequent promise to complete

Reply:
Just wondering how we would use the results from two gets at the same time without a .then for each.